sources/lib/Model/CollectionIterator.php

```diff
@@ -198,6 +199,15 @@ public function slice($name)
         return $this->convertSlice(parent::slice($name), $name);
     }
 
+    /**
+     * @param Closure $callback
```

Comment:
Useless line, it’s already in function prototype. That’s duplicate information and add a chance to modify one and not the other.